config/controller.yaml

```diff
@@ -75,10 +75,10 @@ spec:
 
           # This is gcr.io/google.com/cloudsdktool/cloud-sdk:302.0.0-slim
           "-gsutil-image", "gcr.io/google.com/cloudsdktool/cloud-sdk@sha256:27b2c22bf259d9bc1a291e99c63791ba0c27a04d2db0a43241ba0f1f20f4067f",
-          # The shell image must be root in order to create directories and copy files to PVCs.
-          # gcr.io/distroless/base:debug as of February 17, 2022
+          # The shell image must allow root in order to create directories and copy files to PVCs.
```

Comment:
I'm splitting the shell stuff off because clearly the root requirements haven't been plumbed through for `shell-image` as Scott did for `git-init`, and will chase that separately.

I'm not sure why our KinD-based tests didn't show the PVC test problems using this, which makes me wonder if the e2e test suite is being clever with `t.Skip()` or something (or I'm missing a feature flag).